include/uk/arch/ctx.h

```diff
@@ -56,12 +57,76 @@
 #define UKARCH_CTX_OFFSETOF_SP 8
 #endif
 
+#define	UKARCH_AUXSPCB_SIZE					\
```

Comment:
Actually I would rather not use `_CB` as it would feel inconsistent with `AUXSP` which is joined. But regarding comments, will do. I will also rename `uksc`->`uk_sysctx`.